docs/source/markdown/podman-image-sign.1.md

```diff
@@ -36,6 +36,14 @@ Store the signatures in the specified directory.  Default: /var/lib/containers/s
 
 Override the default identity of the signature.
 
+#### **--authfile**=*path*
+
+Path of the authentication file. If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using docker login.
+
+Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using podman login.
```

Comment:
Oops, one more: I find this wording confusing. First, the `, which` in line 41 has an ambiguous referent and could be rewritten to make it clearer. Second, I really don't understand the chain of precedence. Is the default line auth.json, or config.json, or what? I would suggest looking at this with fresh eyes.

Reply:
Removed to match podman create.